helm/aqua-app-scanner/templates/scanner-deployment.yaml

```diff
@@ -23,44 +23,20 @@ spec:
         app: {{ .Release.Name }}-scanner
       name: {{ .Release.Name }}-scanner
     spec:
-    {{- if .Values.serviceAccount }}
       serviceAccount: {{ .Values.serviceAccount }}
-    {{- else }}
-      serviceAccount: {{ .Release.Name }}-sa
-    {{- end }}
       containers:
       - name: scanner
         securityContext:
           {{- toYaml .Values.securityContext | nindent 12 }}
         image: "{{ .Values.repositoryUriPrefix }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: "{{ .Values.image.pullPolicy }}"
-        {{- if .Values.secretName }}
-        env:
-        - name: SCANNER_USER
-          valueFrom:
-            secretKeyRef:
-              name: {{ .Values.secretName }}
-              key: {{ .Values.userKey }}
-        - name: SCANNER_PASSWORD
-          valueFrom:
-            secretKeyRef:
-              name: {{ .Values.secretName }}
-              key: {{ .Values.passwordKey }}
-        {{- end}}
```

Comment:
I presume this is replaced by this? https://github.com/giantswarm/aqua-app/pull/43/files?file-filters%5B%5D=.tpl&file-filters%5B%5D=.yaml#diff-c35ce371536ef76f30532c44b7cad5d8d65a10041db0e3b22eb0c37a1a730a5eR51-R53

Reply:
That's my presumption as well.